### DISCUSSION TITLE
Display Profile Override

### DISCUSSION THREAD

**Discussion by KratorD** [zikula/core#5366] [Q&A]
Hi all!

I'm using Zikula 3.0 and Profile extension for profiles. Now I'd like to change the profile display page, getting new data from certain modules of my own and displaying it on the page. What's the best way to do this? Is there a way to "override" this? Or do I just modify the profile module?

Thanks for all!

**Comment by Guite** [comment]
Just override the template in your theme and call some custom Twig functions in order to add/select your custom data.

**Reply by KratorD** [reply]
Oh, wow. I thought it would be something colorful, a file in a path or something similar (like you do with templates). I even thought about using Event Subscribers, or hooks. That's why I didn't know which option was the "logical" one.

Last night I was thinking that, since Profile is not going to have any updates for Zikula 3, the best thing would be to modify Profile without any problem and everything would be easier that way.

**Comment by KratorD** [comment]
Thanks Guite for all
